Project-1/README.md

```diff
@@ -13,63 +13,226 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
+  (d)  O that says that x is an only child 
+  O= parent_of (∃parent_of=1) 
   (e)  T that says that x has exactly two brothers 
+  T= Parent_of: (wx,wy,wz) ∃parent_of.(∃Parent_of=3)^Male(∃Male>=2)
+  Male: (y,z)
   ```
 
 
 5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+B= Male^∃Parent_of.(∃parent_of>=2).
+O= parent_of (∃parent_of=1)
+O^~B
+B^~O
+Jason: B
+(Mary Ann, Jason)
+(Mary Ann, Mona)
+Killian: O
+Amanda: Parent_of Killian
+Jason: (w,B)
+Mary Ann: Z
+Mona: Y
+Killian: (O,x)
+Amanda: v
+parent: (z,v)
+brother: w
+Only child: x
+child: w,y,x
+parent_of: zw,zy,vx
+Male: x,w
 
 6. Explain the difference - using natural language - between the first-order prefixes:
   ```
   (a) ∃x∀y and ∀x∃y
+  The first claims that for something such that it is x, every thing such that it is Y has a certain relationship with it. The second claims that for all things such that they are X, there is some Y that has a certain relationship with it.
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  The first claims that for something such that it is x, everything such that it is Y has a certain relationship with X and that there is something such that it is Z has a certain relationship with all Ys and some X. The second claims that for all things such that they are X, there is something such that it is Y that has a certain relationship with it and everything such that it is Z has a certain relationship with some Y and everything such that it is X.
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  The first claims that for all things such that they are x, there is something such that is is Y in a certain relationship with it and everything that is such that it is z is in a certain relationship with all xs and some y and there is some w which is in a certain relationship with all zs and some y and all xs. In the second, it claims that there is something such that it is x where there all ys are in a certain relationship with it, and there is some z which has a certain relationship with some x and all ys and all things such that they are w are in a certain relationship with some z and all ys and some x.
 ```
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+X1>Y1, X2>Z1<Y2, X2>Y2 Arrows indicate relationship R and its directionality
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+X1>Y1>Z1, X2>Z2
 ```
 	
 8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
   ```
   (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+1.¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+2.¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))(1)
```

Comment:
What rule justifies moving from ¬∀x∀y to ¬∀y? That is, what rule justifies instantiating x to a within the scope of negation without first removing the negation?

Reply:
Unfortunately it looks like the generator included does not cite any rules, so I cannot directly speak to what it itself had in mind.

With that said, I believe what it did there was to move a few steps at once. A negation of a universal quantifier is equivalent to an existential negation (something is not the thing in question). With that in mind, showing an instantiation is false would be an instance of an existential negation being true. As such, it skipped the intermediary stage of switching the universal with a existential quantifier and moving the negation in a step. It also skipped the step of moving from the existential to a specific instance of the formula, resulting in a move from a formula to an instance of it in one move.